linera-service/src/linera/main.rs

```diff
@@ -669,10 +669,9 @@ impl Runnable for Job {
 
                 for rpc_msg in &proposals {
                     if let RpcMessage::BlockProposal(proposal) = rpc_msg {
-                        let (executed_block, _) =
-                            WorkerState::new("staging".to_string(), None, storage.clone())
-                                .stage_block_execution(proposal.content.block.clone())
-                                .await?;
+                        let executed_block = context
```

Comment:
Can we make the worker state private now?

Reply:
I don't think we can make `WorkerState` private (or rather `pub(crate)`) yet. It still seems to be used (`linera-rpc`, `linera-sdk` for tests, and `linera-service` still has one usage in the client).